content/english/datastruct/Stacks/stack.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Stack Implementation"
+date: 2020-02-10T13:24:17-07:00
+draft: false
+weight: 2
+--- 
+
+<p style="text-align: center;"><iframe width="60%" height="600px" src="https://www.youtube.com/embed/K6PWO0Mn0Jo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+<link rel="stylesheet" href="../../style.css">
+
+![stackExample](../../img/stackExample.png)
+
+## Stacks
+
+<hr>
+
+A stack is eerily similar to a queue. The only difference are the method names and the manner in which we retrive the items.
+
+Before we create a stack, we will have to import the stack class. To make life easier, we can use a * instead to import all. It will look something like this.
```

Comment:
```suggestion
Before we create a stack, we will have to import the stack class. To make life easier, we can use a `*` instead to import all. It will look something like this.
```